src/transformers/chai-assert.test.js

```diff
@@ -0,0 +1,174 @@
+/* eslint-env jest */
+import chalk from 'chalk';
+import { wrapPlugin } from '../utils/test-helpers';
+import plugin from './chai-assert';
+
+chalk.enabled = false;
+
+const wrappedPlugin = wrapPlugin(plugin);
+let consoleWarnings = [];
+beforeEach(() => {
+    consoleWarnings = [];
+    console.warn = v => consoleWarnings.push(v);
+});
+
+function testChanged(msg, source, expectedOutput) {
+    test(msg, () => {
+        const result = wrappedPlugin(source);
+        expect(result).toBe(expectedOutput);
+        expect(consoleWarnings).toEqual([]);
+    });
+}
+
+testChanged('does not change if chai is not imported',
+`
+// @flow
+assert.equal(foo, bar, baz);
+`,
+`
+// @flow
+assert.equal(foo, bar, baz);
+`
+);
+
+testChanged('does not change if assert is not imported from chai',
+`
+// @flow
+import { should } from 'chai';
+`,
+`
+// @flow
+import { should } from 'chai';
+`
+);
+
+testChanged('does not change if assert is not required from chai',
+`
+// @flow
+const should = require('chai').should;
+`,
+`
+// @flow
+const should = require('chai').should;
+`
+);
+
+const mappings = [
+  ['assert.fail(foo, bar, baz);', 'expect(false).toBe(true);'], // TODO: ?
```

Comment:
Ah yes, the infamous "I forgot"...

I guess it's a question of: is this good enough? It does the same thing, but jest doesn't really have anything that means the same semantically.